static/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -37,3 +37,68 @@ Once initialized in a project, DVC populates its installation directory
 - `.dvc/updater.lock` - a lock file for `.dvc/updater`.
 
 - `.dvc/lock` - a lock file for the whole dvc project.
+
+## Structure of cache directory
+
+The structure of directory depends if the data is stored in a file or is in a
+directory.
+
+In case of a file, it is converted to a MD5 checksum which is a 32 characters
+long string. The first two characters are assigned to name the directory inside
+`.dvc/cache` and rest are given to name the cache file. For example, if a data
+file, say `Posts.xml.zip`, is converted to a MD5 checksum, it will evaluate to
+`ec1d2935f811b77cc49b031b999cbf17`. The cache file for this data file will be
+stored as `.dvc/ec/1d2935f811b77cc49b031b999cbf17` on the local storage and if
+it is pushed to a remote storage, its location will be
+`<prefix>/ec/1d2935f811b77cc49b031b999cbf17` where prefix is the name of the
+remote storage. `/tmp/dvc-storage` can be one example of a prefix.
+
+For the second case, let us consider a directory of 2 images.
+
+```dvc
+$ tree
+.
+├── cat.jpeg
+└── index.jpeg
+```
+
+On running `dvc add` on this directory of images, a `Dvcfile` is created with
```

Comment:
`dirname.dvc` file is created by default, or you can simply specify DVC-file or `.dvc` files.